CMakeLists.txt

```diff
@@ -211,7 +211,19 @@ target_link_libraries(openPMD::thirdparty::nlohmann_json
 
 # external library: toml11
 if(openPMD_USE_INTERNAL_TOML11)
-    add_subdirectory("${openPMD_SOURCE_DIR}/share/openPMD/thirdParty/toml11")
+    # EXCLUDE_FROM_ALL ensures that toml11 is not part of
+    # make install.
+    # The library is header-only and linking it against
```

Comment:
To be more precise: the library is private and not used in our public API interfaces (aka no types from it are used in our header files) :)